apps/condo/domains/notification/schema/PushToken.test.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Generated by `createschema notification.PushToken PushToken`
+ */
+
+const { makeLoggedInAdminClient, makeClient, UUID_RE, DATETIME_RE } = require('@core/keystone/test.utils')
+
+const { PushToken, createTestPushToken, updateTestPushToken } = require('@condo/domains/notification/utils/testSchema')
+const { expectToThrowAccessDeniedErrorToObjects, expectToThrowAuthenticationErrorToObjects } = require('@condo/domains/common/utils/testSchema')
+
+/**
+ * 1. Регистрация токена
```

Comment:
Cyrillic detected! I don't understand it %)

Reply:
fixed